modules/developer-cli-odo-basic-odo-cli-commands.adoc

```diff
@@ -391,6 +391,30 @@ Push source code to a component.
   odo push my-component --context ~/mycode
 ----
 
+== registry
+
+Create and modify custom registries.
+
+.Example using registry
```

Comment:
Is registry in this context a command? If so, I'd write "Example using the registry command". Otherwise, add an article: "Example using a/the registry"